tables/portals.txt

```diff
@@ -3149,6 +3149,9 @@ dew_fild01 48 65 dew_dun02 302 30
 dew_dun02 305 30 dew_fild01 50 65
 
 # Lasagna
+izlude 198 213 lasagna 206 297 2000 c c r0 n
```

Comment:
I updated a couple of portals on rock ridge yesterday along with lasagna.
For iRO from izlude to lasagna is:

izlude 195 213 lasagna 206 297 2000 c c r0 n